src/seedu/addressbook/data/person/Address.java

```diff
@@ -35,9 +35,8 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
-    @Override
```

Comment:
Note that replacing this might affect some other function calls that try to print the Address object via System.out.println(<Address object>) or System.out.println(<Address object>.toString())